histogram/types.go

```diff
@@ -10,19 +12,32 @@ type Centroid struct {
 type HistogramGranularity int8
 
 const (
-	MINUTE HistogramGranularity = iota
+	SECOND HistogramGranularity = iota
```

Comment:
Remove this? Shouldn't be exporting this externally as we only support minute, hour and day bins.

Reply:
@laullon Will you be removing this?

Reply:
I use it for unit testing... will try to find another way, and then remove it